docs/design-proposals/non_evm_inbound_memo/2024_10_02_standard_memo_format.md

```diff
@@ -0,0 +1,131 @@
+| **Optional**    | No (0b000 for now)              | No (0b00 or 0b01 or 0b10)      | The operation code is already differentiated by method names in contract based chains |
+
+
+### `byte-2`: Data Flags
+
+|                 | bit 7         | bit 6        | bit 5         | bit 4             | bit 3            | bit 2            | bit 1             | bit 0            |
+|-----------------|---------------|--------------|---------------|-------------------|------------------|------------------|-------------------|------------------|
+| **Name**        | flag reserved | flag reserved| flag reserved | flag RevertMessage| flag AbortAddress| flag CallOnRevert| flag RevertAddress| flag Payload     |
+| **Optional**    | --            | --           | --            | Yes               | Yes              | Yes              | Yes               | Yes              |
+<br>
+
+Supported `operation codes` :
+
+The `operation code` is designed for the developers to explicitly specify the intention of an inbound transaction in non-contract chains (e.g. Bitcoin, Doge, …). A 3-bit operation code would allow `8` operation codes at maximum.
+
+| 0b000      | 0b001            | 0b010      |
+|------------|------------------|------------|
+| deposit    | deposit_and_call | call       |
+
+<br>
+
+
+Supported `encoding flags`:
+This flag is to tell `zetaclient` how the memo fields should be decoded. A 2-bit weight flag seems good enough and won’t change in future.
+
+| 0b00         | 0b01             | 0b10             |
+|--------------|------------------|------------------|
+| ABI encoded  | lite-data short  | lite-data long   |
+<br><br>
+
+
+## 4. The types
+
+To decode/encode the memo fields with the lite-date format, we've defined a few types below.
+To decode/encode the memo fields with ABI format, just follow the [ABI Spec](https://docs.soliditylang.org/en/develop/abi-spec.html).
+
+
+|                Field        | Size (byte)  | Description                                                                                                                     |
+|-----------------------------|--------------|---------------------------------------------------------------------------------------------------------------------------------|
+| **ZEVM address**            | fixed, 20    | The raw bytes representation of a ZEVM address. ZEVM address is the same as standard Ethereum address.                          |
+| **non-EVM address**         | variable     | The raw bytes representation of a non-EVM external chain address. Different addresses have different lengths, e.g., Bitcoin vs Solana; Bitcoin Segwit vs Legacy. |
+| **bytes**                   | variable     | The raw bytes of arbitrary data.                                                                                                |
+<br><br>
+
+
+## 5. Byte layout of variable-length types
+
+There are two ways to encode your the memo fields into raw bytes. The `ABI encoded` and `lite-data` encoded.
```

Comment:
Any link what is lite-data?

Reply:
Seems introducing unexplained name will confuse reader. Let me just use the adjective `compact` to replace `lite-data`